sources/Inputs/src/NetworkManager.cpp

```diff
@@ -108,8 +108,16 @@ NetworkManager::buildTree() {
       auto pmax = (*it_g)->getPMax();
       if ((*it_g)->isVoltageRegulationOn() && (DYN::doubleEquals(-targetP, pmin) || -targetP > pmin) && (DYN::doubleEquals(-targetP, pmax) || -targetP < pmax)) {
         // We don't use dynamic models for generators with voltage regulation disabled and an active power reference outside the generator's PQ diagram
```

Comment:
Is this comment still relevant ?

Reply:
This comment is to explain the if condition, and why it is here.